scripts/src/view/input/fugaziInput.tsx

```diff
@@ -116,21 +117,37 @@ export class FugaziInputView extends base.SuggestibleInputView<FugaziInputProper
 	}
 
 	protected onSuggestionItemPressed(item: statements.Statement): void {
-		let newValue: string = "",
+		let oldValueWords: string[] = this.getValue().split(" "),
+			expressionsIterator = (item.getExpression() as input.CommandExpression).getExpressions().getIterator(),
+			newValue: string = "",
 			haveStoppedForParameter = false,
 			iterator = item.getRule().getTokens().getIterator();
```

Comment:
maybe rename this `iterator`.  there's `expressionsIterator`, so this one should also have a more explicit name no?